test_util/test_aws_vpc.py

```diff
@@ -191,20 +154,41 @@ def main():
     vpc = None  # Set if the test owns the VPC
 
     if options.host_list is None:
-        log.info('CCM_VPC_HOSTS not provided, requesting new VPC from CCM...')
-        vpc = make_vpc(use_bare_os=options.test_install_prereqs)
-        host_list = vpc.hosts()
+        log.info('VPC_HOSTS not provided, requesting new VPC ...')
+        random_identifier = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(10))
+        unique_cluster_id = "installer-test-{}".format(random_identifier)
+        log.info("Spinning up AWS VPC with ID: {}".format(unique_cluster_id))
+        if options.test_install_prereqs:
+            os_name = "cent-os-7"
+        else:
+            os_name = "cent-os-7-dcos-prereqs"
+        # TODO(mellenburg): Switch to using generated keys
+        bw = test_util.aws.BotoWrapper(
+            region=DEFAULT_AWS_REGION,
+            aws_access_key_id=options.aws_access_key_id,
+            aws_secret_access_key=options.aws_secret_access_key)
+        vpc = test_util.aws.VpcCfStack.create(
+            stack_name=unique_cluster_id,
+            instance_type='m4.xlarge',
+            instance_os=os_name,
+            instance_count=5,  # 1 bootstrap, 1 master, 2 agents, 1 public agent
+            admin_location='0.0.0.0/0',
+            key_pair_name='default',
+            boto_wrapper=bw)
+        vpc.wait_for_stack_creation()
+
+        vpc_info = vpc.get_vpc_host_ips()
+        log.info('AWS provided VPC info: ' + repr(vpc_info))
+        host_list = [i.public_ip for i in vpc_info]
     else:
         host_list = options.host_list
 
-    assert os.path.exists('ssh_key'), 'Valid SSH key for hosts must be in working dir!'
-    # key must be chmod 600 for test_runner to use
-    os.chmod('ssh_key', stat.S_IREAD | stat.S_IWRITE)
-
     # Create custom SSH Runnner to help orchestrate the test
     ssh_user = 'centos'
-    ssh_key_path = 'ssh_key'
+    ssh_key_path = 'default_ssh_key'
```

Comment:
Since this code is no longer responsible for generating the key, would it make sense to pass the key's path as a config parameter rather than hardcoding it?